examples/getting-started/README.md

```diff
@@ -0,0 +1,99 @@
+
+```bash
+git clone https://github.com/HubSpot/cms-react.git
+```
+
+CMS React projects have a 3 significant directories:
+
+- `getting-started-project`: The HubSpot project directory that gets uploaded via the Projects system to build and deploy your CMS React code found in the `/src` directory.
+- `getting-started-project/src`: A package inside your project directory containing all your assets, styles, components, modules, islands, and any local development tooling you need, such as ESLint, Prettier, TypeScript, etc.
+- `getting-started-theme`: This is where your HubL templates and theme configuration lives.
+
+At the root of this example, you will configure any local dev tooling you want, such as ESLint, Prettier, TypeScript, etc.
+
+### 2. Install Dependencies
+
+Navigate to the `cms-react/examples/getting-started` directory in your terminal and run:
+
+```bash
+npm install
+```
+
+This will install all the local development tooling you need, including `@hubspot/cli`, `@hubspot/cms-dev-server`, and any other dev tooling you configured.
+
+### 3. Local Development
+
+To run this example project locally, navigate to the `getting-started-project/src` directory and run:
+
+```bash
+npm run start
+```
+
+Then navigate to [http://hslocal.net:3000](http://hslocal.net:3000) to see an index page that links to all your modules.
+
+### 4. Uploading and Deploying on HubSpot
+
+Run `hs init` from the root of the repository to configure it to upload to your HubSpot account using your personal access key. When setting up JavaScript assets for the first time, you will need to deactivate and regenerate your personal access key, ensuring it includes `CMS Pages`, `Design Manager`, `Developer Projects`, and `GraphQL Data Fetching` permissions. This will create a `hubspot.config.yaml` file required for both uploading changes and local development. See the [HubSpot CLI documentation](https://developers.hubspot.com/docs/cms/guides/getting-started-with-local-development) for more information.
+
+> **Warning:** If you are using an existing access key from a previous call to `hs init` or `hs auth`, you will need to deactivate and regenerate the access key to include new scopes necessary for local CMS React development.
+
+Double-check you are still in `getting-started/` and run:
+
+```bash
+npm run deploy
+```
+
+This will upload the `getting-started-project` to your HubSpot account. Once uploaded, built, and deployed, you can add the modules and partials to your HubSpot templates.
+
+From the repository root, run:
+
+```bash
+npm run upload:hubl
+```
+
+This uploads the corresponding example HubL files to your account. You should now be able to create a page from one of the `getting-started-theme` templates and see the output of the React components in the page preview.
+
+### 5. Previewing Local Changes on Proxied Pages
+
+You can preview your local CMS React components inside live HubL-rendered pages. To do this, create a page:
+
+- Go to Control Center
```

Comment:
"Control Center" I think is an internal term - though probably littered in these docs... maybe refer to it as "Website Pages" in HubSpot.